app/src/debug/res/values/ids.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="visible" type="id"/>
```

Comment:
Without these IDs defined the layout preview doesn't display correctly in Android Studio. They aren't necessary at runtime, it seems to make more sense to restrict them to debug builds only.